app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
While you did a job implementing this. It would be nice if you added foreign keys. Kindly set up assoiciation correctly according to the ERD diagram you were provided with. Kindly have a look [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

Here is a hint on how you can do this:

has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'